functions/paperChannelTenders/src/index.ts

```diff
@@ -1,3 +1,43 @@
-import { apiHandler } from './handlers/api-handler';
+import handlerRoute from './routes/tender-routes';
+import { validatorEvent } from './middlewares/validators';
 
```

Comment:
Aggiungere TSdoc e in config.ts leggere le configurazioni dalle varibili d'ambiente con un default, come fatto in https://github.com/pagopa/pn-bff/blob/develop/functions/senderDashboardDataIndexer/src/app/config.js

Reply:
resolved